src/js/components/reviewData/ReviewDataNarrative.jsx

```diff
@@ -81,11 +104,28 @@ export default class ReviewDataNarrative extends React.Component {
     }
 
     render() {
+        const onKeyDownHandler = createOnKeyDownHandler(this.downloadCommentsFile.bind(this));
```

Comment:
To prevent the binding from happening every time `render` is invoked, you can move the binding to the constructor:
`this.downloadCommentsFile = this.downloadCommentsFile.bind(this);`
and use `this.downloadCommentsFile` here.

Reply:
Should I do the same for these?
`this.changeFile.bind(this)`
`this.textChanged.bind(this)`
`this.saveNarrative.bind(this)`

Reply:
That would be ideal - optional for the scope of this PR!